csrc/sparse/cutlass/common.hpp

```diff
@@ -0,0 +1,33 @@
+#pragma once
```

Comment:
nit: maybe move this to `cutlass_extensions` to avoid someone writing duplicate code in the future